api/src/opentrons/calibration_storage/ot2/models/v1.py

```diff
@@ -32,8 +32,8 @@ class TipLengthModel(BaseModel):
         default_factory=CalibrationStatus,
         description="The status of the calibration data.",
     )
-    uri: typing.Union[LabwareUri, Literal[""]] = Field(
-        ..., description="The tiprack URI associated with the tip length data."
+    definitionHash: str = Field(
+        ..., description="The tiprack hash associated with the tip length data."
```

Comment:
If we're modifying this v1 model in-place instead of making a v2, can we leave a comment like this for posterity?

```suggestion
    # Old data may have a `uri` field, replaced later by `definitionHash`.
    # uri: typing.Union[LabwareUri, Literal[""]] = Field(
    #    ..., description="The tiprack URI associated with the tip length data."
    # )
    definitionHash: str = Field(
        ..., description="The tiprack hash associated with the tip length data."
```